tests/test_vision_explanation.py

```diff
@@ -196,38 +200,142 @@ def test_vision_explain_loadmodel():
+                                                  0,
+                                                  threshold=threshold,
+                                                  num_masks=num_masks)
+
+        if not num_masks > 0:
+            with pytest.raises(
+                    ValueError,
+                    match='Number of masks parameter not a \
+                             positive int.'):
+                salient_scores = pg.pointing_game(img_fname,
+                                                  0,
+                                                  threshold=threshold,
+                                                  num_masks=num_masks)
+
+        salient_scores = pg.pointing_game(img_fname,
+                                          0,
+                                          threshold=threshold,
+                                          num_masks=num_masks)
+        overlap = pg.calculate_gt_salient_pixel_overlap(salient_scores,
+                                                        gt_bbox)
+
+        # calculated salient pixel overlap equals expected brute force
+        good = 0
+        total = 0
+        for iindex, i in enumerate(salient_scores[0]):
+            for jindex, j in enumerate(i):
+                if j > 0:
+                    if (gt_bbox[1] <= iindex <= gt_bbox[3]
+                       and gt_bbox[0] <= jindex <= gt_bbox[2]):
+                        good += 1
+                if (gt_bbox[1] <= iindex <= gt_bbox[3]
+                   and gt_bbox[0] <= jindex <= gt_bbox[2]):
+                    total += 1
+        overlap_check = good/total
+
+        assert round(overlap, 2) == round(overlap_check, 2)
+
+    def test_vision_explain_evaluation():
+        """End to end testing for explanation evaluation."""
+        # pointing game run
+        imgpath = os.path.join('python', 'vision_explanation_methods',
+                               'images', '128.jpg')
+        # load fastrcnn model
+        modelpath = os.path.join('python', 'vision_explanation_methods',
+                                 'models', 'fastrcnn.pt')
+        # save tested result in res
+
+        # use helper function from above to fetch model
+        _ = download_assets(modelpath)
+
+        # run the main function for saliency map generation
+        device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+        model = _get_instance_segmentation_model(5)
+        model.load_state_dict(torch.load(modelpath, device))
+        model.to(device)
+        model.eval()
+        detection_model = PytorchDRiseWrapper(model=model,
+                                              number_of_classes=1000)
+
+        # find saliency scores for top 20% of salient pixels
+        # do this for the second object in an image
+        pg = PointingGame(detection_model)
+        index = 0
+        s = pg.pointing_game(imgpath, index)
+
+        # check that the saliency map exists and has 3 channels
+        assert (len(s) == 3)
```

Comment:
```suggestion
        assert len(s) == 3
```
Nit: don't need brackets